crdppf/lib/content.py

```diff
@@ -1,12 +1,61 @@
+# -*- coding: UTF-8 -*-
+
 from copy import deepcopy
+import ast
+
+from crdppf.lib.wmts_parsing import wmts_layer
+from crdppf.lib.geometry_functions import get_feature_bbox, get_print_format, get_feature_center
+
+
+import logging
+
+log = logging.getLogger(__name__)
 
 def get_content(idemai, request):
+    """ TODO....
+        Explain how the whole thing works...
+    """
+
+    # Configure the WMTS background layer
+    url = request.registry.settings['wmts_getcapabilities_url']
+    defaultTiles =  ast.literal_eval("{"+request.registry.settings['defaultTiles']+"}")
+    layer = defaultTiles['wmtsname']
+    wmts_layer_ = wmts_layer(url, layer)
+
+    # Get the raw feature BBOX
+    bbox = get_feature_bbox(idemai)
 
+    # Get the feature center
+    feature_center = get_feature_center(idemai)
+
+    # Get the print BOX
+    print_box = get_print_format(bbox, request.registry.settings['pdf_config']['fitratio'])
+
+    log.warning('Calling feature: %s' % request.route_url('get_property')+'?ids='+idemai)
```

Comment:
from the crdppf point of view you can get only one property at a time - so id would suffice... or am I missing the point?